docs/docs/graphql.md

```diff
@@ -1504,9 +1504,9 @@ Limit the number of aliases in a document. Defaults to 15.
 
 ##### Example
 
-Aliases allow you rename the data that is returned in a query’s results. They manipulate the structure of the query result that is fetched from your service, displaying it according to your web component's needs.
+Aliases allows you to rename the data that is returned in a query’s results. They manipulate the structure of the query result that is fetched from your service, displaying it according to your web component's needs.
```

Comment:
I think "allow" was correct but definitely need the "to":

```suggestion
Aliases allow you to rename the data that is returned in a query’s results. They manipulate the structure of the query result that is fetched from your service, displaying it according to your web component's needs.
```

Reply:
Fixed